package.json

```diff
@@ -103,9 +113,13 @@
     "rollup-plugin-babel": "^4.0.3",
     "rollup-plugin-commonjs": "^9.2.0",
     "rollup-plugin-node-resolve": "^3.4.0",
-    "rollup-plugin-size-snapshot": "0.7.0",
+    "rollup-plugin-size-snapshot": "^0.7.0",
+    "rollup-plugin-typescript": "^1.0.1",
     "storybook-readme": "^5.0.5",
     "tinycolor2": "^1.4.1",
+    "ts-jest": "^24.1.0",
```

Comment:
We can remove this and just rely on `@babel/preset-typescript` (we're not configuring `jest` to use `ts-jest` in this branch).

Reply:
done